cmd/podman/README.md

```diff
@@ -1,23 +1,6 @@
-# Adding a podman V2 commands
+# Podman CLI
 
-## Build podman V2
-
-```shell script
-$ cd $GOPATH/src/github.com/containers/libpod/cmd/podmanV2
-```
-If you wish to include the libpod library in your program,
-```shell script
-$ go build -tags 'ABISupport' .
-```
-The `--remote` flag may be used to connect to the Podman service using the API.
-Otherwise, direct calls will be made to the Libpod library.
-```shell script
-$ go build -tags '!ABISupport' .
-```
-The Libpod library is not linked into the executable.
-All calls are made via the API and `--remote=False` is an error condition.
-
-## Adding a new command `podman manifests`
+## Adding a new command `podman manifest`
```

Comment:
It would be nice to have a little explanatory text up here.    Something along the lines of:

The following is an example of how to add a new command to the Podman CLI.  In the example, we're showing how the `podman manifest` command was first added to the GitHub repository.